data/mods/blazemod/blaze_test.json

```diff
@@ -235,7 +235,7 @@
         "price" : 1500000,
 		"watertight": true,
 		"seals": true,
-		"contains": 0
+		"contains": 1
```

Comment:
The author is maintaining a separate tree that he isn't PRing.
While removing some functionality that is important for a mod to function should be discussed first, we aren't obliged in any way to support hacks and badly specified items. Containers with `"contains": 0` are badly defined as there is no reason to have a container with no ability to contain.

The change here is just to suppress a warning. The proper fix would be to make it not a container, but the mod in mainline is heavily outdated anyway so there is no point in trying to fix it.
The best way would be to either get someone to semi-regularly PR the updates to the mod or get rid of the mainlined version of the mod to lose the responsibility to maintain a very outdated version of the mod.